src/index.js

```diff
@@ -5,20 +5,23 @@ import { getDnsPromise } from './dns';
 
 function setupTimeoutCapture(wrapperInstance) {
   const { modifiedContext, sendReport, config } = wrapperInstance;
-  var endTime = 599900; /* Maximum execution: 100ms short of 5 minutes */
-  if (config.timeoutWindow < 1) {
+  const { getRemainingTimeInMillis = () => 0 } = modifiedContext;
+
+  // if getRemainingTimeInMillis returns a very small number, it's probably a local invoke (sls framework perhaps)
+  if (config.timeoutWindow < 1 || getRemainingTimeInMillis() < 10) {
     return undefined;
   }
 
-  if (config.timeoutWindow > 0 && modifiedContext.getRemainingTimeInMillis) {
-    endTime = Math.max(
-      0,
-      modifiedContext.getRemainingTimeInMillis() - config.timeoutWindow
-    );
-  }
+  const maxEndTime = 299900; /* Maximum execution: 100ms short of 5 minutes */
```

Comment:
AWS will be increasing to 10 minutes _very_ soon, so maybe we should think about our assumptions here. At the very least, maybe leave it as is?